src/test/kotlin/note/mothers/NoteMother.kt

```diff
@@ -17,6 +17,12 @@ class NoteMother {
             description = null
         )
 
+        fun getValidNoteFrom(title: String, description: String?) = Note(
```

Comment:
I think we should keep the title and descriptions parameters inside the mothers, and the tests will only have to request the desired instance to them (making the mothers responsable for all the correct and incorrect titles/descriptions, so, if we want to change the domain rules some day we will only have to change the mothers and not all the tests).

Therefore, I think in your use case the instantiations needed that aren't implemented at the mothers are:
```kotlin
fun getDifferentValidTitleForNote(note: Note): Note {
      // Here you can use the convenient methods from TitleMother (see my comment about that in the TitleMother.kt review)
}
fun getDifferentValidCommentForNoteWithAnExistingComment(note: Note): Note {
      // Here you can use the convenient methods from DescriptionMother
}
fun getValidCommentForNoteWithoutAnExistingComment(note: Note): Note {
      // Here you can use the convenient methods from DescriptionMother.
}
```